docs/html/user_guide.rst

```diff
@@ -1306,6 +1306,238 @@ issue tracker`_ if you believe that your problem has exposed a bug in pip.
+   Downloading cup-3.18.0-py2.py3-none-any.whl (393 kB)
+   |████████████████████████████████| 393 kB 22.1 MB/s
+   Downloading cup-3.17.0-py2.py3-none-any.whl (382 kB)
+   |████████████████████████████████| 382 kB 23.8 MB/s
+   Downloading cup-3.16.0-py2.py3-none-any.whl (376 kB)
+   |████████████████████████████████| 376 kB 27.5 MB/s
+   Downloading cup-3.15.1-py2.py3-none-any.whl (385 kB)
+   |████████████████████████████████| 385 kB 30.4 MB/s
+   INFO: pip is looking at multiple versions of this package to determine
+   which version is compatible with other requirements.
+   This could take a while.
+   Downloading cup-3.15.0-py2.py3-none-any.whl (378 kB)
+   |████████████████████████████████| 378 kB 21.4 MB/s
+   Downloading cup-3.14.0-py2.py3-none-any.whl (372 kB)
+   |████████████████████████████████| 372 kB 21.1 MB/s
+   Downloading cup-3.13.1-py2.py3-none-any.whl (381 kB)
+   |████████████████████████████████| 381 kB 21.8 MB/s
+   This is taking longer than usual. You might need to provide the
+   dependency resolver with stricter constraints to reduce runtime.
+   If you want to abort this run, you can press Ctrl + C to do so.
+   Downloading cup-3.13.0-py2.py3-none-any.whl (374 kB)
+
+In the above sample output, pip must downloaded multiple versions of
+package cup - cup-3.22.0 to cup-3.13.0 - to find a version that will be
+compatible with the other packages - ``spoon``, ``hot-water``, ``cup``
+etc.
+
+These multiple ``Downloading cup-version`` lines shows pip backtracking.
+
+Possible ways to reduce backtracking occuring
+---------------------------------------------
+
+It's important to mention backtracking behaviour is expected during a
+``pip install`` process. What pip is trying to do is complicated - it is
+working through potentially millions of package versions to identify the
+comptible versions.
+
+There is no guaranteed solution to backtracking but you can reduce it -
+here are a number of ways.
+
+.. _1-allow-pip-to-complete-its-backtracking:
+
+1. Allow pip to complete its backtracking
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In most cases, pip will complete the backtracking process successfully.
+It is possible this could take a very long time to complete - this may
+not be the preferred option.
+
+However there is a possibility pip will not be able to find a set of
+compatible versions.
+
+If you'd prefer not to wait, you can interrupt pip (ctrl and c) and use
+constraints to reduce the number of package versions it tries.
+
+.. _2-reduce-the-versions-of-the-backtracking-package:
+
+2. Reduce the versions of the backtracking package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If pip is backtracking more than you'd like, the next option is to
+constrain the number of package versions it tries.
+
+A first good candidate for this constraining is the package(s) it is
+backtracking on (e.g. in the above example - ``cup``).
+
+You could try:
+
+``pip install tea cup > 3.13``
```

Comment:
```suggestion
``pip install tea "cup > 3.13"``
```

Without quotes, this will be a shell redirection, not a constraint. That's really confusing when it happens (I know, I've done it myself many times!)